packages/compiler-cli/test/ngtsc/env.ts

```diff
@@ -6,19 +6,21 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {CustomTransformers, defaultGatherDiagnostics, Program} from '@angular/compiler-cli';
+import {CustomTransformers, defaultGatherDiagnostics, Program, readConfiguration} from '@angular/compiler-cli';
+import {NgTscPlugin} from '@angular/compiler-cli/src/ngtsc/tsc_plugin';
```

Comment:
```suggestion
import {NgTscPlugin} from '../../src/ngtsc/tsc_plugin';
```

and needs to be moved down probably.